src/plugins/discover/public/application/angular/context/query/actions.tsx

```diff
@@ -67,25 +79,25 @@ export function QueryActionsProvider(Promise) {
     return Promise.try(() =>
       fetchAnchor(indexPatternId, anchorId, [_.fromPairs([sort]), { [tieBreakerField]: sort[1] }])
     ).then(
-      (anchorDocument) => {
+      (anchorDocument: AnchorHitRecord) => {
         setLoadedStatus(state)('anchor');
         state.rows.anchor = anchorDocument;
         return anchorDocument;
       },
-      (error) => {
+      (error: Error) => {
         setFailedStatus(state)('anchor', { error });
         getServices().toastNotifications.addDanger({
           title: i18n.translate('discover.context.unableToLoadAnchorDocumentDescription', {
             defaultMessage: 'Unable to load the anchor document',
           }),
-          text: <MarkdownSimple>{error.message}</MarkdownSimple>,
```

Comment:
think this should be resolved like here:

https://github.com/elastic/kibana/blob/8b8d4d83b6f8808985ee1bcd326fc2d99e47c0e4/src/plugins/visualize/public/application/utils/use/use_visualize_app_state.tsx#L109-L109

If the error message contains e.g. a link in markdown format, this code converts it to be clickable.